backend/routes/flightStatus.js

```diff
@@ -22,7 +23,9 @@ try {
     const response = await amadeus.schedule.flights.get({
         carrierCode, flightNumber, scheduledDepartureDate
     });
-    res.json(JSON.parse(response.body));
+    const flightData = JSON.parse(response.body);
+    const status = calculateFlightStatus(flightData);
```

Comment:
You used to have this calculation in the frontend, right? Are you planning to remove it?

Reply:
Yeap, I'm removing the status check from frontend. I'll add the frontend changes in next pull request so it's easier to follow.